llama_stack/providers/remote/vector_io/chroma/chroma.py

```diff
@@ -56,7 +56,9 @@ async def add_chunks(self, chunks: List[Chunk], embeddings: NDArray):
             )
         )
 
-    async def query(self, embedding: NDArray, k: int, score_threshold: float) -> QueryChunksResponse:
+    async def query(
+        self, embedding: NDArray, query_str: None, k: int, score_threshold: float, search_mode: None
```

Comment:
one challenge with changing the ordering of the args here (`query_str` in particular) is that if any user was using the method without the named args, this change will break them. 

Maybe use is limited to server but it's technically breaking change.

Reply:
Yeah, changing the ordering made sense from an API perspective. I can modify them to get added towards the end. It would a breaking change either ways, but at least would not cause confusion in determining types for users with unnamed params.